### ISSUE TITLE
Selective export and extra format

### ISSUE THREAD

**Issue by eltrentin** [tildaslash/RatticWeb#369]
Would be interesting to have the possibility to:
- export "only selected" password (not all password of the group)
- export password in .CSV format
- export / print password as a PDF

**Comment by rayvenshire** [comment]
Are you talking about adding these features into the UI?

You can get all of this from poking at the database directly.

**Comment by eltrentin** [comment]
Exactly. Would be interesting to add this feature on the UI to permit a
instant and easy export. It s an alternative to the keepass export mode

Inviato da Samsung Mobile
Il 22/giu/2015 22:34, "Mike Ray" <notifications@github.com> ha scritto:
